src/test/regress/sql/multi_behavioral_analytics_basics.sql

```diff
@@ -12,9 +12,8 @@ FROM (
     FROM users_table AS u,
          events_table AS e
     WHERE u.user_id = e.user_id
-      AND u.user_id >= 10
-      AND u.user_id <= 25
-      AND e.event_type IN (100, 101, 102)
+      AND u.user_id = 1
```

Comment:
I think we are losing some tests here. For us `>=` and `<=` are not same with` =`. Do you have any reason to convert them?

Reply:
This was what I worried. Okay, I will find another way around